### PR TITLE
Fix calendar icon

### PR DESCRIPTION
**Pull request by go-bears**
this is a messy commit. apologizes! i tried rebasing to clean it up but failed.  The commit that matters is the one that fixes the calendar. The other two are superfluous. I think you can do merge & squash as a project maintainer? hoping to get better at git!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,10 +114,17 @@ <h2 style="text-align:center;">Currently in use at</h2>
                     <a href="http://www.calpoly.edu"><li class="col"><img src="assets/calpoly.svg" class="greydout" alt="CalPoly" height="60" width="72"></li></a>
                     <a href="http://www.berkeley.edu"><li class="col"><img src="assets/Berkeley.svg" class="greydout" alt="Berkeley" height="60" width="147"></li></a>
                     <a href="http://www.sheffield.ac.uk/"><li class="col"><img src="assets/sheffield.svg" class="greydout" alt="Sheffield" height="60" width="137"></li></a>
+<<<<<<< HEAD
```
Comment (minrk) [inline index.html]: Looks like you've committed some conflict markers. Can you remove these and resolve the conflicts?

Comment (minrk) [inline index.html]: Sorry, it looks like you still have the merge markers in this file. Can you do:

```
git fetch origin
git checkout origin/master index.html
git commit -m 'restore index.html to master'
```

to reset it back to master?

**Comment by rgbkrk** [comment]
Looks like you've got it. I'll merge and squash like was said earlier in the thread.

**Comment by go-bears** [comment]
whew! thanks so much for the guidance @rgbkrk !

**Comment by rgbkrk** [comment]
Thanks to you!

**Pull request merged**